Vagrantfile

```diff
@@ -59,12 +59,15 @@ Vagrant.configure(2) do |config|
   end
 
 
-
-
+  ansible_tags = ENV["MINERVA_VAGRANT_ANSIBLE_TAGS"] || false
 
   config.vm.provision "ansible" do |ansible|
     ansible.verbose = "vvvv"
 
+    if ansible_tags
+        ansible.tags = "minerva-update"
+    end
+
```

Comment:
Can we make this more general?  i.e.,  have MINERVA_VAGRANT_ANSIBLE_TAGS expect a comma separated list of tags that get passed directly to ansible.tags?